etc/adminhtml/system/adyen_testing_performance.xml

```diff
@@ -55,5 +55,14 @@
                 ]]>
             </comment>
         </field>
+        <field id="cleanup_old_webhooks" translate="label" type="select" sortOrder="50" showInDefault="1" showInWebsite="1" showInStore="1">
+            <label>Clean-up old webhooks</label>
+            <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+            <config_path>payment/adyen_abstract/cleanup_old_webhooks</config_path>
+            <tooltip>
+                Webhooks older than certain days will be removed from the database by a cronjob if this feature is enabled.
+                The default value is 90 days and this can be configured by overriding `payment/adyen_abstract/required_days_old_webhooks` configuration path.
```

Comment:
Are we then hiding `required_days_old_webhooks `?

Reply:
Hello @acampos1916,

The initial idea was hiding this field in order to prevent accidents caused by set this value less than a legitimate value. We were only expecting advanced users to change this value through database if they absolutely need it.

However, I've realised we are also removing the flexibility of this setup after reading your comment. Let's discuss this comment internally with the product team before merging this PR.

Cheers!